app/stac_api/validators.py

```diff
@@ -187,3 +187,27 @@ def validate_item_properties_datetimes(
             properties_start_datetime,
             properties_end_datetime,
         )
+
+
+def validate_json_payload(serializer):
+    '''
+    Validate JSON payload and raise error, if extra payload or read-only fields
+    in payload are found
+    Args:
+        serializer: serializer for which payload is checked
+
+    Raises:
+        ValidationError if extra or read-only fields in payload are found
+    '''
+    # see: https://www.django-rest-framework.org/api-guide/serializers/#object-level-validation
```

Comment:
If you mean the comment pointing to the drf doc, I think it still does make sense here.
It points to parts of docu that explain overriding of validate() and also accessing `.initial_data`.
So it might be useful from my point of view.